src/Client.php

```diff
@@ -41,11 +41,29 @@ class Client
     private static $wireVersionForReadConcern = 4;
     private static $wireVersionForWritableCommandWriteConcern = 5;
 
+    /**
+     * @var \MongoDB\Driver\Manager
+     */
```

Comment:
These type annotations for private members should be removed. These shouldn't appear in generated documentation and userland code should not interact with them.